cinema_service/urls.py

```diff
@@ -1,9 +1,11 @@
 from django.contrib import admin
 from django.urls import path, include
+from django.conf import settings
+from django.conf.urls.static import static
 
 urlpatterns = [
     path("admin/", admin.site.urls),
     path("api/cinema/", include("cinema.urls", namespace="cinema")),
     path("api/user/", include("user.urls", namespace="user")),
     path("__debug__/", include("debug_toolbar.urls")),
```

Comment:
Consider including the `debug_toolbar` URL pattern conditionally, only in development environments. This can be done by checking the `DEBUG` setting before including the Debug Toolbar URLs.